frontend/packages/console-shared/src/components/actions/menu/ActionMenuItem.tsx

```
@@ -67,6 +67,7 @@ const ActionItem: FC<ActionMenuItemProps & { isAllowed: boolean }> = ({
     onClick: handleClick,
     'data-test': label,
```

Biome (2.5.0)

[error] This property is later overwritten by an object member with the same name.

(lint/suspicious/noDuplicateObjectKeys)